scrapper.py

```diff
@@ -19,10 +20,17 @@ def get_talks(response):
 
 if __name__ == "__main__":
     response = get_html(url="https://www.pylight.org")
-    meetup_details = check_date_and_address(response)
-    talks = get_talks(response)
-    print(meetup_details)
-    print("\n")
-    for talk in talks:
-        print(talk.text)
-        print("\n")
+    raw_details = check_date_and_address(response)
+    raw_talks = get_talks(response)
+
+    talks = []
+    for talk in raw_talks:
+        talk_details = talk.text.split('\n')
+        talks.append(dict(speaker=talk_details[0], title=talk_details[1]))
```

Comment:
Tę pętlę dałoby się napisać jako [list comprehension](https://www.pythonforbeginners.com/basics/list-comprehensions-in-python). No i co w przypadku, kiedy `talk_details` okaże się listą jednoelementową? ;)

Reply:
Jeżeli nie trzeba rozbijać `talk_details` to można to napisać jako list comprehension. Przy rozbiciu na `speaker` i `title` celowo zrobiłem pętle aby było bardziej przejrzyste.